tests/e2e/README.md

```diff
@@ -1,137 +1,23 @@
-# Frontend UI testing
+# GiveWP e2e testing
 
-The frontend UI testing tests whether the contents of Give's features display and function as intended. These tests don't focus on the aesthetics but rather tests the correct output on the webpage and the interactions with it.
+End-to-end testing (e2e) checks whether GiveWP's features display and function as intended. These tests focus on the output of the webpage and interactions with it.
 
 ## How to run the tests?
 
 ### Prerequisites
-These tests run inside a dockerized container, so it is necessary that you have the [Docker Engine](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed on your system.
+These tests run inside a dockerized container, so it is necessary that you have the [Docker Engine](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed on your system. For a quick installation from scratch, you can simply install the [Docker Desktop app](https://www.docker.com/products/docker-desktop).
 
 A sample `wordpress.sql` is provided within the `/sample-data/` folder that has a few sample forms, donations, and donors to test in various combinations.
 
-### Manual Testing
-After setting up the local development environment, running tests manually is fairly simple. Navigate to the Give root folder where the `docker-compose.yml` file resides and set up the container by running:
+### Testing
 
-```sh
-docker-compose up -d
-```
+After setting up the local development environment, running tests manually is fairly simple. Navigate to the Give root folder and get things started by running:
 
-Now that the container is ready, run the tests by firing the following commad:
-```sh
-npm run test
-```
+```npm run test:e2e```
 
-By default this will run the tests in the headless mode.
-If you wish to run the tests in non-headless mode, you can set `headless: false` inside `tests/e2e/jest-puppeteer.config.js`.
-This will launch an instance of the Chrome browser and the tests will begin.
+This will use `wp-env` to spin up a new local version of WordPress with your current changes to GiveWP. Once setup, Cypress is then opened, and ready to run tests against it.
 
-You can also change the speed of execution by setting `slowMo: 50`. The ideal speed range is between 30-80. _Low number indicates high speed_.
+When you're done with testing, simply close the Cypress GUI, and run `npm run wp-env stop` to stop the WP instance that was created.
 
-### Automation testing
```

Comment:
All this content is pretty informative for our old tests. @henryholtgeerts are we planning on writing similar content in our Gitbook?

Reply:
Yep! A version of this information, made more specifically relevant to our new e2e testing methodology, will be included in the GitBook. I opted to add a link to the GitBook, rather than expand on the info too much in this README, since the GitBook will be more up to date and comprehensive.